### ISSUE TITLE
Labels and group on Overlay

### ISSUE THREAD

**Issue by basnijholt** [ioam/holoviews#58]
```
kwargs = {'key_dimensions':[B_dim], 
          'value_dimensions':[E_dim], 
          'extents':bounds}

hv.Overlay.from_values([hv.Curve(zip(x, y[i]), **kwargs) for i in range(10)])
```
produces:
![download 5](https://cloud.githubusercontent.com/assets/6897215/7550648/38c31e4e-f66a-11e4-843d-1fc4472d4d9a.png)

```
kwargs = {'key_dimensions':[B_dim], 
          'value_dimensions':[E_dim], 
          'extents':bounds,
          'label':'Test label'
          'group':'Test group'}

hv.Overlay.from_values([hv.Curve(zip(x, y[i]), **kwargs) for i in range(10)])
```
produces:
![download 4](https://cloud.githubusercontent.com/assets/6897215/7550652/3cac994a-f66a-11e4-9371-abbf0a420837.png)

**Comment by jlstevens** [comment]
Thank you for the bug report!

I believe Philipp recently pushed a fix for this bug - have you tried the latest version of master? We plan on having a bug-fix release very soon!

**Comment by basnijholt** [comment]
I've just updated to the latest master, but the bug is still there.

**Comment by jlstevens** [comment]
Yes sorry - we were already aware of the bug but we don't have a fix just yet!

**Comment by philippjfr** [comment]
Yes, this is pretty annoying and I swear this working at one point. I'll investigate it tonight and hopefully push a fix.

**Comment by philippjfr** [comment]
Fixed in f2018a2ccab814e, was caused by sanitization happening after rather than before finding a unique path.
